results-tabulation-api/ext/ExtendedElection/ExtendedElectionProvincialCouncilElection2021/TallysheetTemplates/administrative_district_result_candidates.py

```diff
@@ -0,0 +1,62 @@
+from orm.entities import Template
```

Comment:
atleast for our convenience, we may need to come up with codes for those who doesn't have one. Otherwise it's difficult for us to manage. But, we could not show those codes in the user interfaces as required by the EC

Reply:
what was the format you followed for creating codes?

Reply:
Actually there was no strict rule. Whatever make sense for us